docs/api/primary-elements.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Commonly used data types [WIP]
+sidebar_position: 4
+hide_title: false
+description: Building blocks of data and their APIs.
+vega_network: TESTNET
+---
+
+While there are hundreds of available endpoints, the categories below cover the basics to get you started with understanding what you need to interact with Vega. 
+
+## Parties
+A party is a single user, defined as a Vega public key. As one person or entity can have many public keys, this is a unique identifier as far as an individual key's actions.  A party ID and a Vega public key (pubkey) are equivalent, and can be used interchangeably.
```

Comment:
Saying a party ID and a Vega public key are equivalent here sounds like they are 2 different things that can be used to do the same action, while they are actually just 2 names for the same thing.